lib/editor/components/timetable/TripSeriesModal.js

```diff
@@ -0,0 +1,67 @@
+// @flow
+// $FlowFixMe: Flow doesn't know about useState: https://stackoverflow.com/questions/53105954/cannot-import-usestate-because-there-is-no-usestate-export-in-react-flow-with
+import React, {useState} from 'react'
+import {Button, Modal} from 'react-bootstrap'
+
+import * as tripActions from '../../actions/trip'
+import {getComponentMessages} from '../../../common/util/config'
+import HourMinuteInput from '../HourMinuteInput'
+import type {Trip} from '../../../types'
+
+type Props = {
+  addNewTrip: typeof tripActions.addNewTrip,
+  constructNewRow: (trip: ?Trip, tripSeriesStartTime: number) => ?Trip,
+  onClose: () => void,
+  show: boolean
+}
+
+const TripSeriesModal = (props: Props) => {
+  const [startTime, setStartTime] = useState(0)
+  const [headway, setHeadway] = useState(0)
+  const [endTime, setEndTime] = useState(0)
+
+  const messages = getComponentMessages('TripSeriesModal')
+
+  const onClickGenerate = () => {
+    const {addNewTrip, constructNewRow, onClose} = props
+    for (let time = startTime; time <= endTime; time += headway) {
+      addNewTrip(constructNewRow(null, time))
+    }
+    setStartTime(0)
+    setEndTime(0)
+    setHeadway(0)
+    onClose()
+  }
+
+  const {Body, Footer, Header, Title} = Modal
+  const {onClose, show} = props
+  const generateTripsDisabled = (!startTime || !endTime || !headway) || (startTime > endTime)
+  return (
+    <Modal show={show} onHide={onClose}>
+      <Header><Title>{messages('createTripSeriesQuestion')}</Title></Header>
+      <Body>
+        {messages('createTripSeriesBody')}
+        <br />
```

Comment:
```suggestion
        <p>{messages('createTripSeriesBody')}</p>
```
I think we should avoid break tags

Reply:
Changed in [0d258e6](https://github.com/ibi-group/datatools-ui/pull/892/commits/0d258e683ef66e3c08bc2cb25f47eafa001a3489)